charts/aws-fsx-csi-driver/templates/node-daemonset.yaml

```diff
@@ -1,32 +1,38 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: {{ include "helm.fullname" . }}-daemonset
+  name: fsx-csi-node
   labels:
-    {{- include "helm.labels" . | nindent 4 }}
+    {{- include "aws-fsx-csi-driver.labels" . | nindent 4 }}
 spec:
   selector:
     matchLabels:
-      {{- include "helm.selectorLabels" . | nindent 6 }}-daemonset
+      app: fsx-csi-node
```

Comment:
Adding this will cause upgrade issues since matchLabels is immutable. It looks like maybe it is being added to ensure that is continues to be included on the kustomize manifests. If that is the case, I would move if from here to an else in the selectorLabels helper template.